lib/manageiq/floe/workflow/runner/podman.rb

```diff
@@ -3,6 +3,32 @@ module Floe
     class Workflow
       class Runner
         class Podman < ManageIQ::Floe::Workflow::Runner
+          def run!(resource, env = {}, secrets = {})
+            raise ArgumentError, "Invalid resource" unless resource&.start_with?("docker://")
+
+            image = resource.gsub("docker://", "")
+
+            secret_guid = nil
+
+            if secrets && !secrets.empty?
+              require "securerandom"
```

Comment:
Should probably move this to initialize or something (or just require all the time), so it's not in the hot path.

Reply:
Done in 06e64f8